pages/vi/profiles/nachiketdhamankar.md

```diff
@@ -0,0 +1,13 @@
+# Nachiket Dhamankar
+
+:house: Boston, MA <br />
```

Comment:
For icons it is better to copy the actual icon by copy paste instead of tag so that it can be even viewed on raw.githack link

Reply:
Didn't know that the emoji wouldn't be rendered. Fixed that.